### ISSUE TITLE
Orca `estimator.fit()` with ray backend need the `initial_epoch` parameter

### ISSUE THREAD

**Issue by ForJadeForest** [intel-analytics/BigDL#7293]
There is `initial_epoch` parameter in estimator.fit() with spark backend, that can help the scheduler  to remember how the lr change in the resuming training model. But in ray backend, there is not the parameter with the same effect. 
I think it's better to add the `initial_epoch` in the ray backend to help reload the scheduler status.

**Comment by hkvision** [comment]
@sgwhat Take a look, can we directly add this?

**Comment by hkvision** [comment]
@lalalapotter Please check this issue :)

**Comment by hkvision** [comment]
For ray backend, initial epoch always starts from 0, we need to support specifying this argument.
For spark backend, initial epoch is unchanged, and when fitting twice, the initial epoch may not be correct?

**Comment by lalalapotter** [comment]
After checking the TensorFlow [document](https://www.tensorflow.org/api_docs/python/tf/keras/Model#fit) and some tests, the behaviors are different between ray backend estimator and TensorFlow Models when fitting twice. For example, we use the following code in TensorFlow:

```python
model.fit(epochs=2, ...)
# output:
# epoch 1/2
# epoch 2/2
model.fit(epochs=2, ...)
# output:
# epoch 1/2
# epoch 2/2
```
When we use the similar setting in ray backend estimator, the behaviors are different.
```python
estimator.fit(epochs=2, ...)
# output:
# epoch 1/2
# epoch 2/2
estimator.fit(epochs=2, ...)
# output:
# epoch 3/4
# epoch 4/4
```
In `TFRunner`, we will update the current `self.epoch` after fitting, therefore, the estimator will resume the former fitting epochs when fitting twice. @yangw1234 Can you help to check the related [code](https://github.com/intel-analytics/BigDL/blob/f583287e01ca8ffe5a5f1a12ea94631a8b01a49f/python/orca/src/bigdl/orca/learn/tf2/tf_runner.py#L395-L404)? Will it affect the behavior of Horovod if we change the code?

**Comment by yangw1234** [comment]
> After checking the TensorFlow [document](https://www.tensorflow.org/api_docs/python/tf/keras/Model#fit) and some tests, the behaviors are different between ray backend estimator and TensorFlow Models when fitting twice. For example, we use the following code in TensorFlow:
> 
> ```python
> model.fit(epochs=2, ...)
> # output:
> # epoch 1/2
> # epoch 2/2
> model.fit(epochs=2, ...)
> # output:
> # epoch 1/2
> # epoch 2/2
> ```
> 
> When we use the similar setting in ray backend estimator, the behaviors are different.
> 
> ```python
> estimator.fit(epochs=2, ...)
> # output:
> # epoch 1/2
> # epoch 2/2
> estimator.fit(epochs=2, ...)
> # output:
> # epoch 3/4
> # epoch 4/4
> ```
> 
> In `TFRunner`, we will update the current `self.epoch` after fitting, therefore, the estimator will resume the former fitting epochs when fitting twice. @yangw1234 Can you help to check the related [code](https://github.com/intel-analytics/BigDL/blob/f583287e01ca8ffe5a5f1a12ea94631a8b01a49f/python/orca/src/bigdl/orca/learn/tf2/tf_runner.py#L395-L404)? Will it affect the behavior of Horovod if we change the code?

I guess it should not affect Horovod, but maybe you should test it to make sure.
